python/torch_mlir/eager_mode/torch_mlir_dispatch.py

```diff
@@ -0,0 +1,258 @@
+            arg, arg_dtype = tensor_kwargs[i]
+            annotations.append(TorchTensorType(shape=tuple(arg.shape), dtype=arg_dtype))
+            tensor_kwargs_flat.append(arg)
+
+    return annotations, tensor_args, tensor_kwargs_flat
+
+
+def fix_inplace(schema, out, all_tensor_args):
+    qual, name = schema.name.split("::")
+    op = OP_REGISTRY.get_by_triple((qual, name, schema.overload_name))
+    if op.is_mutable:
+        if len(op.returns) > 1:
+            raise RuntimeError("TorchMLIR doesn't handle multiple aliased returns yet.")
+        else:
+            aliased_arg = next(
+                arg
+                for arg in op.arguments
+                if "alias_info" in arg and arg["alias_info"]["is_write"]
+            )
+            # TODO: over-engineered
+            assert (
+                "alias_info" in op.returns[0]
+                and op.returns[0]["alias_info"]["is_write"]
+                and len(op.returns[0]["alias_info"]["after"]) == 1
+                and op.returns[0]["alias_info"]["after"][0]
+            )
+            assert (
+                len(aliased_arg["alias_info"]["after"]) == 1
+                and aliased_arg["alias_info"]["after"][0]
+                == op.returns[0]["alias_info"]["after"][0]
+            )
+            np.copyto(all_tensor_args[0], out)
+
+    return out
+
+
+def try_torch_mlir_eager(op, args, kwargs, backend):
+    if hasattr(op, "op_name"):
+        op_name = op.op_name
+    elif hasattr(op, "__name__"):
+        # handle builtin_function_or_method
+        op_name = op.__name__
+    else:
+        raise RuntimeError(f"op {op} has no name")
+
+    if op_name == "detach":
+        raise RuntimeError("detaching")
+    if "layout" in kwargs and kwargs["layout"] is not None:
+        # handle ops like ones_like, which comes in with a sparse strided layout
+        # TODO: remove when torch-mlir handles different layouts
+        warnings.warn(
+            f"{kwargs['layout']} layout not supported; changing to default layout."
+        )
+        kwargs["layout"] = None
+    if "memory_format" in kwargs and kwargs["memory_format"] is not None:
+        warnings.warn(
+            f"{kwargs['layout']} memory format not supported; changing to default memory format."
+        )
+        kwargs["memory_format"] = None
+
+    if not hasattr(op, "_schema"):
+        raise RuntimeError(f"op {op} has no schema.")
+
+    script_fun = build_script_function(op._schema, args, kwargs)
+    annotations, tensor_args, tensor_kwargs_flat = annotate_args_kwargs(
+        script_fun, args, kwargs
+    )
+
+    eager_module = build_module(script_fun, annotations)
+    with eager_module.context:
+        pm = PassManager.parse(
+            "torch-function-to-torch-backend-pipeline,torch-backend-to-linalg-on-tensors-backend-pipeline"
+        )
+        pm.run(eager_module)
+    compiled = backend.compile(eager_module)
+    jit_module = backend.load(compiled)
```

Comment:
jit_module isn't a great name. Maybe loaded_module (and call the above "compiled" as "compiled_module" instead).